app/services/battle_report_service.rb

```diff
@@ -1,6 +1,7 @@
 class BattleReportService < ApplicationService
   class BattleReportValidationError < StandardError; end
   MINIMUM_ELAPSED = 300.freeze
+  MINIMUM_RANKED_ELAPSED = 1800.freeze
```

Comment:
Stomps under 30 min should count no?

Reply:
It's basically impossible for a game to be that short without a drop is my understanding, and that probably should count for ranking